frontends/web/src/components/alert/Alert.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h } from 'preact';
+import i18n from '../../i18n/i18n';
+import Dialog from '../dialog/dialog';
+import { Button } from '../forms';
+
+var alerted: (message: string) => void;
+
+interface State {
+    message: string;
+    active: boolean;
+}
+
+interface Alert {
+    button: any;
+}
+
+class Alert extends Component<{}, State> {
+    constructor(props) {
+        super(props);
+        alerted = this.alerted;
+        this.state = {
+            message: '',
+            active: false,
+        };
+    }
+
+    componentDidMount() {
+        document.addEventListener('keydown', this.handleKeyDown);
+    }
+
+    componentWillUnmount() {
+        document.removeEventListener('keydown', this.handleKeyDown);
+    }
+
+    handleClose = e => {
```

Comment:
I wanted to activate `--noImplicitAny` (see https://www.typescriptlang.org/docs/handbook/compiler-options.html) but could not because of the `checkJS`. However, once I can, `e` would be marked as an error because it is implicitly `any`. I haven't found the optimal solution yet but so far I fixed it with https://github.com/digitalbitbox/bitbox-wallet-app/blob/f4309109de5cf8392d2aba990e1798b6c041ff9c/frontends/web/src/components/fiat/fiat.tsx#L33-L37 Because you don't need e here in this case, you can omit the `as` part here.